response_objects/run_single.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import annotations
+
+from gamedata import FileParser
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from runs import RunParser
+
+class RunResponse:
+    def __init__(self, parser: FileParser, keys: dict[str, int], *, prev_char: RunParser = None, next_char: RunParser = None, autorefresh: bool, redirect: bool):
+        self.run = parser
+        self.path_as_list = list(parser.path)
```

Comment:
Shouldn't need to do `list(parser.path)` here anymore, as it now returns a list by default.